consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/converter/XmlToObjectConverter.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.moj.sdt.cmc.consumers.converter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import java.io.IOException;
+import org.springframework.stereotype.Component;
```

Comment:
Looks like the imports may need to be re-ordered - I think java imports are normally separated from the others.

Reply:
done